environments/magento2.darwin.yml

```diff
@@ -21,5 +21,16 @@ services:
       - .${WARDEN_WEB_ROOT:-}/pub/media:/var/www/html/pub/media:delegated
       - appdata:/var/www/html
 
+  crontab:
+    volumes:
+      - /var/run/docker.sock:/var/run/docker.sock
```

Comment:
This should likely be marked as read-only `:ro`. Also with the docker.sock being mounted, this container will need to be marked as privileged or it will not work on Linux installations with SELinux or AppArmor enabled.

Reply:
My assumption is that it uses `/var/run/docker.sock` not only to listen but also to trigger exec of the images - but I don't understand enough of the docker API to know that for sure.